README.md

```diff
@@ -53,7 +53,13 @@ As there will be differences between local development and running on DGX (for e
 
 Note: The values present in the template config files are examples, you can remove any except those in `[server]` and `[project]` which are necessary for MLOps. Outside of these you are encouraged to add and modify the config files as relevant to your project.
 
-### 2. `project/Network.py`
+### 2. `project/XNATDataImport.py`
```

Comment:
I think this should stay to be part of the DataModule component.

Reply:
As it is its own component, I think it should have its own explanation.

Will create an issue to make a code structure/architecture image.